modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,296 +1,140 @@
-+
-You can now use the following syntax to connect all datasets in a dataverse:
-+
-----
-CONNECT LINK (dataverse_name.)? Local (, (dataverse_name.)? Local)*
-----
-
-[#deprecation-600]
-=== Deprecated Platforms
-
-Support for the following platforms will be removed in a future release:
+= Release Notes for Couchbase Server 6.5 Beta
 
-* CentOS 6
-* macOS 10.11 (El Capitan)
-* Red Hat Enterprise Linux 6
-* SUSE Linux Enterprise Server (SLES) 11
-* Ubuntu 14.04
-* Windows Server 2012
+Couchbase Server 6.5 Beta brings many sought-after relational capabilities to the NoSQL world. This release targets improving developer agility, reduced TCO, enhanced manageability, and provides enterprise-grade security and performance. 
 
+include::ROOT:partial$beta-disclaimer.adoc[tag=admonition]
 
-[#supported-platforms-600]
-=== New Supported Platforms
-
-There are no new supported platforms in this release.
-
-See xref:install:install-platforms.adoc[Supported Platforms] for the complete list of supported platforms.
-
-[#known-issues-600]
-=== Known Issues
+Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
-==== Administration/Cluster Management
+// [#release-650]
+// == Release 6.5
 
-[#table_knownissues_v600-cluster,cols="25,66"]
-|===
-| Issue | Description
+Couchbase Server 6.5 Beta was released in August 2019.
 
-| https://issues.couchbase.com/browse/MB-23074[MB-23074^]
-| *Summary*: Performance issues may be observed when running Couchbase Server on CentOS 7.3 with kernel 3.10.0-514.6.
-|===
+[#changes-in-behavior-650]
+=== Major Changes in Behavior from Previous Releases
 
-==== Analytics Service
+This section notes major changes in behavior from previous releases.
 
-[#table_knownissues_v600-analytics,cols="25,66"]
-|===
-| Issue | Description
+==== Backup and Restore
 
-| https://issues.couchbase.com/browse/MB-30637[MB-30637^]
-| *Summary:* Backup and restore for the Analytics service artifacts using the `cbbackupmgr` utility is currently not supported.
+===== Compression
 
-| https://issues.couchbase.com/browse/MB-30007[MB-30007^]
-| *Summary*: It is possible for a CONNECT BUCKET statement to fail temporarily with the error message "The vbucket belongs to another server(0x7)" when rebalancing Data nodes.
+In the previous versions of Couchbase Server, value compression always occurred on the backup client. 
 
-*Workaround*: Retry the operation.
+Starting with version 6.5, `cbbackupmgr` will leverage Server compression and backup the documents as compressed whenever possible. If the data is compressed by default, it will be backed up as compressed. If the data is not compressed, it will be compressed first and then backed up when value compression is opted. This improves the performance by reducing the size of the dataset to backup for transmission on the pipe and for the backup itself. 
```

Comment:
```suggestion
Starting with version 6.5, `cbbackupmgr` will leverage Server compression and back up the documents as compressed whenever possible.
If the data is compressed by default, it will be backed up as compressed.
If the data is not compressed, it will be compressed first and then backed up when value compression is opted. This improves the performance by reducing the size of the data set to back up for transmission on the pipe and for the backup itself. 
```